gunicorn/arbiter.py

```diff
@@ -186,12 +188,17 @@ def init_signals(self):
         # initialize all signals
         for s in self.SIGNALS:
             signal.signal(s, self.signal)
-        signal.signal(signal.SIGCHLD, self.handle_chld)
+
+    def write_message_to_pipe(self, one_byte_message):
+        try:
+            os.write(self.PIPE[1], one_byte_message.to_bytes(1))
```

Comment:
```suggestion
            os.write(self.PIPE[1], one_byte_message.to_bytes(1, byteorder='big'))
```

On pre-3.11 Pythons, the signature is:

```python
to_bytes(length, byteorder, *, signed=False)
```

I.e. there was no default for the `byteorder` argument.

Reply:
Thanks for the heads up! :+1: I have incorporated your suggested change in the next change-set!